dftimewolf/lib/state.py

```diff
@@ -12,6 +13,7 @@
 from dftimewolf.lib import utils
 from dftimewolf.lib.modules import manager as modules_manager
 
+logger = logging.getLogger('dftimewolf')
```

Comment:
I would arge to use here:

```
...getLogger('dftimewolf.state')
```

and that way you can have a more granularity into where the logs are coming from

Reply:
Gotcha. In this case, I don't think it makes much difference for the user to know that this is coming from the state (I'm OK with conflating the `state` with dftimewolf's main execution). WDYT?

Reply:
it's also for debugging purposes, that is for you to know that the error came up in wha tmodule/part.... your call though

Reply:
Let's leave it as-is for now, if this is something that I need to change later on I will.